packages/discv5/README.md

```diff
@@ -14,29 +14,25 @@ Included is a libp2p peer-discovery compatibility module.
 #### Example
 
 ```typescript
-import { Discv5Discovery, ENR } from "@chainsafe/discv5";
-import Libp2p from "libp2p";
+import { Discv5Discovery } from "@chainsafe/discv5";
+import {createLibp2p} from "libp2p";
 import PeerId from "peer-id";
 
 const myPeerId: PeerId = ...;
 
-const bootstrapEnrs: ENR[] = [...];
+const bootEnrs: ENR[] = [...];
```

Comment:
can you add an import for this type?

Reply:
we have a general issue with our readme examples also in the Lodestar repo, might wanna speed some time investigating https://github.com/ChainSafe/lodestar/issues/6300. This would ensure the examples are runnable and detect issues due to code refactoring / dependency updates.